girder/molecules/molecules/utilities/molecules.py

```diff
@@ -80,13 +77,7 @@ def create_molecule(data_str, input_format, user, public, gen3d=True,
         if input_format == 'cjson':
             cjson = json.loads(data_str)
```

Comment:
This is not a new issue to this PR, but I just noticed that if the input format is cjson, it gets loaded here, but then it immediately gets overwritten below with `cjson = json.loads(avogadro.convert_str(data_str, input_format, 'cjson'))`. This essentially runs the cjson through avogadro2 and gets it back out.

I am not sure if we want to do this or not, but this would probably be fixed in a separate PR.

Reply:
I did notice that, I can do a follow up PR, I was try to preserve the existing logic here.